### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by darrenodi**

### CODE REVIEW

**Review by mke2111 — COMMENTED** [review]
### Hey @darrenodi, 👋🏽  

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+linters-config/html-css/.github/workflows/linters.yml
+@SpaYco
+SpaYco Bump up outdated packages (#142)
+Latest commit dfd8253 on Apr 15
+ History
+ 3 contributors
+@MauricioRobayo@SpaYco@bolah2009
+48 lines (45 sloc)  1.45 KB
+  
+name: Linters
```
Comment (mke2111) [inline .github/workflows/linters.yml:34]: > You have added the linter files, great work.

- [ ] Now it's time to make sure that they are detected in the PR. For this to happen, remove all unnecessary content in the linters.yml file. I recommend you to use the given file and do not attempt to alter a thing in the file.

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
```
Comment (mke2111) [inline README.md:1]: - [ ] I recommend that you use only the templates provided by microverse. Use this [readme template](https://github.com/microverseinc/readme-template) and then customize it to your own liking by adding in the specific sections

File: .github/workflows/linters.yml (hunk 1 above)
Comment (darrenodi) [reply .github/workflows/linters.yml:34]: Please review changes again

File: README.md (hunk 2 above)
Comment (darrenodi) [reply README.md:1]: Yes, I corrected it just now :)

**Comment by darrenodi** [comment]
updated

**Comment by mke2111** [comment]
That's nice. Take your time to make all requested changes. Only when you are through, submit another CRR and a code reviewer will give you a review. 
Best regards. 😄

**Comment by darrenodi** [comment]
Okay got it

**Review by mke2111 — COMMENTED** [review]
### Hey @darrenodi, 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] Please take your time to implement all the changes I requested earlier on. None of them has been implemented yet. Remember to push to github all the changes that you make. Also, you have one review left so only and only submit another CRR once you have done all that's requested.

> If you need help with these implementations, you can reach out to your partners, stand=up team members, or even code reviewers. Do not hesitate. 😄 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by darrenodi** [comment]
Please take a look - https://github.com/darrenodi/hello-microverse/pull/3/

**Pull request merged**